src/app/components/user-card/user-card.component.html

```diff
@@ -0,0 +1,11 @@
+<span #ref><ng-content></ng-content></span>
+<span *ngIf="!ref.innerHTML.trim()">
```

Comment:
Angular 18 supports default content for <ng-content>:

https://www.codemotion.com/magazine/frontend/web-developer/angular-fallback-content-in-ng-content/

It can be replaced with:

`<ng-content><b>User</b></ng-content>`